aboutme.html

```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
+
     <title>About Me</title>
     <link rel="stylesheet" href="homestyle.css">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
   	<header class="navbar">
```

Comment:
You can have this as a header or simply as 
```
<nav>
</nav>
```
for greater semantic meaning in your tags